src/vmm/src/vstate.rs

```diff
@@ -1245,13 +1272,17 @@ mod tests {
         // Stop it by sending exit.
         assert!(vcpu_handle.send_event(VcpuEvent::Exit).is_ok());
 
-        // Validate vCPU thread ends execution.
-        vcpu_handle
-            .join_vcpu_thread()
```

Comment:
I think now we can delete the unused `join_vcpu_thread` method.

Reply:
Thanks. Will do.

Reply:
Addressed in 8ee01ae78c8935b6c29b4a9d67fe815024a68bc2